stories/coverage.fmf

```diff
@@ -0,0 +1,100 @@
+story:
+    As a QE I want to document the test coverage,
+    so that it can be easily tracked which requirement
+    is covered by which test case and vice-versa.
+
+/notes:
+    summary:
+        Story notes
+    story:
+        As a product owner I want to add additional content to
+        the requirements, like notes or context,
+        so that extra information can be provided.
+    description:
+        There should be a parameter that has no strict
+        requirement and can be used to provide any
+        extra information that may be necessary.
+    example: |
+        story:  As an admin I want product A doing B,
+                so that I can use it for C.
+        notes:  The feature should help our customer
+                to achieve D.
+    priority: low
+
+/priority:
+    summary:
+        Story priority
+    story:
+        As a product owner I want to prioritize the requirements,
+        so that the importance can be stated.
+    description:
+        Priority is an important attribute which helps the readers
+        to understand the importance of the story.
+    example: |
+        story:      As an admin I want product A doing B,
+                    so that I can use it for C.
+        priority:   high
+    priority: high
+
+/relevancy:
+    summary:
+        Story relevancy
+    story:
+        As a product owner I want to connect requirements
+        with products or versions, so that I can filter out items
+        relevant (or not relevant) for a particular product.
+    description:
+        Relevancy is an easy way to bind stories
+        to particular product(s).
+    example: |
+        story:      As an admin I want product A doing B,
+                    so that I can use it for C.
+        relevancy:  "distro < fedora-28: False"
+    priority: medium
```

Comment:
I understand the use case. Just note, we're obsoleting relevancy with a slightly improved concept which should cover the use case and some a bit more flexibility. See #101 for the specification outline.

Reply:
Right, I've noticed the relevancy was reimplemented for TMT, but have not checked the details.
Checking it quickly, it should meet this requirement I believe.
I'll update the example to meet the new syntax. Is there anything else to update in the story? It is still called relevancy, right?

Reply:
Good, thanks. Just the proposed name for the section with rules is now `adjust`. I've fixed that.